packages/amazonq/src/lsp/chat/webviewProvider.ts

```diff
@@ -51,25 +53,59 @@ export class AmazonQChatViewProvider implements WebviewViewProvider {
         this.webview = webviewView.webview
 
         const lspDir = Uri.parse(LanguageServerResolver.defaultDir)
+        const dist = Uri.joinPath(globals.context.extensionUri, 'dist')
         webviewView.webview.options = {
             enableScripts: true,
             enableCommandUris: true,
-            localResourceRoots: [lspDir, Uri.parse(path.dirname(this.mynahUIPath))],
+            localResourceRoots: [lspDir, dist],
         }
 
+        const source = 'vue/src/amazonq/webview/ui/amazonq-ui-connector-adapter.js' // Sent to dist/vue folder in webpack.
+        const serverHostname = process.env.WEBPACK_DEVELOPER_SERVER
+        const connectorAdapterPath =
+            serverHostname !== undefined
+                ? Uri.parse(serverHostname)
+                      .with({ path: `/${source}` })
+                      .toString()
+                : webviewView.webview.asWebviewUri(Uri.parse(path.join(dist.fsPath, source))).toString()
         const uiPath = webviewView.webview.asWebviewUri(Uri.parse(this.mynahUIPath)).toString()
-        webviewView.webview.html = await this.getWebviewContent(uiPath)
+        webviewView.webview.html = await this.getWebviewContent(uiPath, connectorAdapterPath)
 
         this.onDidResolveWebviewEmitter.fire()
     }
 
-    private async getWebviewContent(mynahUIPath: string) {
+    private async getWebviewContent(mynahUIPath: string, hybridChatConnector: string) {
+        const featureConfigData = await featureConfig.getFeatureConfigs()
+
+        const isSM = isSageMaker('SMAI')
+        const isSMUS = isSageMaker('SMUS')
+        const disabledCommands = isSM ? `['/dev', '/transform', '/test', '/review', '/doc']` : '[]'
         const disclaimerAcknowledged = AmazonQPromptSettings.instance.isPromptEnabled('amazonQChatDisclaimer')
+        const welcomeCount = globals.globalState.tryGet('aws.amazonq.welcomeChatShowCount', Number, 0)
+
+        // only show profile card when the two conditions
+        //  1. profile count >= 2
+        //  2. not default (fallback) which has empty arn
+        let regionProfile: RegionProfile | undefined = AuthUtil.instance.regionProfileManager.activeRegionProfile
```

Comment:
theres a minor amount of duplicated code between this (serves mynah ui from flare) and the original webviewProvider (serves mynah ui from the local package). Once we fully switch to the language server implementation we can start de-duping this code